libs/server/features/src/account-connection/account-connection.service.ts

```diff
@@ -156,9 +156,7 @@ export class AccountConnectionService implements IAccountConnectionService {
             `Disconnected connection id=${connection.id} type=${
                 connection.type
             } provider_connection_id=${
-                connection.type === 'plaid'
-                    ? connection.plaidItemId
-                    : connection.finicityInstitutionId
+                connection.plaidItemId
```

Comment:
I missed this in my teller implementation, but I think we can just update these all to connection.tellerEnrollmentId, we shouldn't have any plaid or finicity connections moving forward.